### PR TITLE
Externalize JS: icds_reports

### PR DESCRIPTION
**Pull request by orangejenny**
Couldn't find any references to a `fieldset_location_async` element either in the code or in a couple of MPR/ASR reports on india's `icds-test` domain, but this seem straightforward enough to migrate.

@nickpell / @emord

### CODE REVIEW

File: custom/icds_reports/static/icds_reports/js/base.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+hqDefine("icds_reports/js/base", function() {
+    $(function() {
+        $(document).on('change', '#fieldset_location_async select', function(e) {
+            e.stopPropagation();
+            var state = $('select.form-control')[0].selectedIndex;
+            var applyBtn = $('#apply-filters');
+            if (state == 0) {
```
[the remark this answers is not in the document]
Comment (orangejenny) [reply custom/icds_reports/static/icds_reports/js/base.js]: ignoring

**Review by nickpell — APPROVED** [review]
Curious, is this something that was caught by automated scripts or discovered by searching manually?

**Comment by orangejenny** [comment]
@nickpell How the script tag was found? Manual search. I'd like to add a lint rule that checks for inline js but haven't looked much into it. [eslint-plugin-html](https://www.npmjs.com/package/eslint-plugin-html) might be helpful.

**Comment by orangejenny** [comment]
Clicked through some ASR and MPR reports on `icds-test` on india, didn't see any issues.

**Pull request merged**